<?xml version="1.0" encoding="UTF-8"?>
<ns2:test-suite xmlns:ns2="urn:model.allure.qatools.yandex.ru" start="1461615368888" stop="1461615368952">
	<name>failing angularjs homepage</name>
	<title>failing angularjs homepage</title>
	<test-cases>
		<test-case start="1461615368910" stop="1461615368941" status="failed">
			<name>one more todo</name>
			<title>one more todo</title>
			<description></description>
			<labels>
			</labels>
			<parameters>
			</parameters>
			<steps>
				<step start="1461615368913" stop="1461615368914" status="passed">
					<name>Filling data</name>
					<title>Filling data</title>
					<attachments>
					</attachments>
					<steps>
						<step start="1461615368914" stop="1461615368914" status="passed">
							<name>Fill todo text</name>
							<title>Fill todo text</title>
							<attachments>
							</attachments>
							<steps>
							</steps>
						</step>
					</steps>
				</step>
				<step start="1461615368915" stop="1461615368916" status="broken">
					<name>another internal step</name>
					<title>another internal step</title>
					<attachments>
					</attachments>
					<steps>
						<step start="1461615368916" stop="1461615368916" status="broken">
							<name>failed</name>
							<title>failed</title>
							<attachments>
							</attachments>
							<steps>
							</steps>
						</step>
					</steps>
				</step>
			</steps>
			<attachments>
			</attachments>
			<failure>
				<message>Failed: Expected error to test failing step</message>
				<stack-trace>Error: Expected error to test failing step
    at /Users/sbashkyrtsev/projects/jasmine-allure-plugin/test/system/todo-spec-failing.js:35:54
    at /Users/sbashkyrtsev/projects/jasmine-allure-plugin/node_modules/allure-js-commons/runtime.js:18:31
    at /Users/sbashkyrtsev/projects/jasmine-allure-plugin/test/system/todo-spec-failing.js:35:104
    at /Users/sbashkyrtsev/projects/jasmine-allure-plugin/node_modules/allure-js-commons/runtime.js:18:31
    at Object.&lt;anonymous&gt; (/Users/sbashkyrtsev/projects/jasmine-allure-plugin/test/system/todo-spec-failing.js:36:7)
    at /Users/sbashkyrtsev/projects/jasmine-allure-plugin/test/system/node_modules/jasminewd2/index.js:96:23
    at new ManagedPromise (/Users/sbashkyrtsev/projects/jasmine-allure-plugin/test/system/node_modules/selenium-webdriver/lib/promise.js:1048:7)
    at controlFlowExecute (/Users/sbashkyrtsev/projects/jasmine-allure-plugin/test/system/node_modules/jasminewd2/index.js:82:18)
    at TaskQueue.execute_ (/Users/sbashkyrtsev/projects/jasmine-allure-plugin/test/system/node_modules/selenium-webdriver/lib/promise.js:2868:14)
    at TaskQueue.executeNext_ (/Users/sbashkyrtsev/projects/jasmine-allure-plugin/test/system/node_modules/selenium-webdriver/lib/promise.js:2851:21)
From: Task: Run it(&quot;one more todo&quot;) in control flow
    at Object.&lt;anonymous&gt; (/Users/sbashkyrtsev/projects/jasmine-allure-plugin/test/system/node_modules/jasminewd2/index.js:81:14)
    at attemptAsync (/Users/sbashkyrtsev/projects/jasmine-allure-plugin/test/system/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:1819:24)
    at QueueRunner.run (/Users/sbashkyrtsev/projects/jasmine-allure-plugin/test/system/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:1774:9)
    at QueueRunner.execute (/Users/sbashkyrtsev/projects/jasmine-allure-plugin/test/system/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:1762:10)
    at Spec.Env.queueRunnerFactory (/Users/sbashkyrtsev/projects/jasmine-allure-plugin/test/system/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:627:35)
    at Spec.execute (/Users/sbashkyrtsev/projects/jasmine-allure-plugin/test/system/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:353:10)
    at Object.fn (/Users/sbashkyrtsev/projects/jasmine-allure-plugin/test/system/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:2360:37)
    at attemptAsync (/Users/sbashkyrtsev/projects/jasmine-allure-plugin/test/system/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:1819:24)
    at QueueRunner.run (/Users/sbashkyrtsev/projects/jasmine-allure-plugin/test/system/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:1774:9)
    at /Users/sbashkyrtsev/projects/jasmine-allure-plugin/test/system/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:1801:16
From asynchronous test: 
Error
    at Suite.&lt;anonymous&gt; (/Users/sbashkyrtsev/projects/jasmine-allure-plugin/test/system/todo-spec-failing.js:26:3)
    at addSpecsToSuite (/Users/sbashkyrtsev/projects/jasmine-allure-plugin/test/system/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:743:25)
    at Env.describe (/Users/sbashkyrtsev/projects/jasmine-allure-plugin/test/system/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:713:7)
    at jasmineInterface.describe (/Users/sbashkyrtsev/projects/jasmine-allure-plugin/test/system/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:3219:18)
    at Object.&lt;anonymous&gt; (/Users/sbashkyrtsev/projects/jasmine-allure-plugin/test/system/todo-spec-failing.js:1:63)
    at Module._compile (module.js:397:26)
    at Object.Module._extensions..js (module.js:404:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)</stack-trace>
			</failure>
		</test-case>
		<test-case start="1461615368942" stop="1461615368942" status="pending">
			<name>other its() failed, but this one is disabled</name>
			<title>other its() failed, but this one is disabled</title>
			<description></description>
			<labels>
			</labels>
			<parameters>
			</parameters>
			<steps>
			</steps>
			<attachments>
			</attachments>
			<failure>
				<message>This test has not been implemented yet</message>
				<stack-trace></stack-trace>
			</failure>
		</test-case>
	</test-cases>
</ns2:test-suite>